internal/lookout/ui/src/services/lookoutV2/UpdateJobSetsService.ts

```diff
@@ -0,0 +1,104 @@
+import { JobSet } from "models/lookoutV2Models"
+import { ApiJobState, SubmitApi } from "openapi/armada"
+import { getErrorMessage } from "utils"
+
+export interface CancelJobSetsResponse {
+  cancelledJobSets: JobSet[]
+  failedJobSetCancellations: {
+    jobSet: JobSet
+    error: string
+  }[]
+}
+
+export interface ReprioritizeJobSetsResponse {
+  reprioritizedJobSets: JobSet[]
+  failedJobSetReprioritizations: {
+    jobSet: JobSet
+    error: string
+  }[]
+}
+
+export class UpdateJobSetsService {
```

Comment:
The job set API calls now have a separate type, so that we no longer have to pass a `JobService` (which can also talk to version 1 of the Lookout API) to `JobSetsContainer`; this makes it clearer that this component no longer talks to version 1 of the Lookout API (and we will need this type anyway once the "cancel selected" dialog in version 2 of the Lookout UI learns to cancel job sets, in analogy with the existing `UpdateJobsServce` type that it already uses). The code below is from `JobService`, without any changes.